includes/jenkins/master/.bash_login

```diff
@@ -0,0 +1,27 @@
+# bash_login
+
+# ASCII art image
+cat << "EOF"
+                        ___
+                     .-'   `'.
+                    /         \
+                    |         ;
+                    |         |           ___.--,
+           _.._     |0) ~ (0) |    _.---'`__.-( (_.
+    __.--'`_.. '.__.\    '--. \_.-' ,.--'`     `""`
+   ( ,.--'`   ',__ /./;   ;, '.__.'`    __
+   _`) )  .---.__.' / |   |\   \__..--""  """--.,_
+  `---' .'.''-._.-'`_./  /\ '.  \ _.-~~~````~~~-._`-.__.'
+        | |  .' _.-' |  |  \  \  '.               `~---`
+         \ \/ .'     \  \   '. '-._)
+          \/ /        \  \    `=.__`~-.
+          / /\         `) )    / / `"".`\
+    , _.-'.'\ \        / /    ( (     / /
+     `--~`   ) )    .-'.'      '.'.  | (
+            (/`    ( (`          ) )  '-;
+             `      '-;         (-'
```

Comment:
I feel like the kraken is soooo big compared to the ships.
Can't we have a tinier cuter octopus?
Like this one maybe?
```
          ,---.
         ( @ @ )
          ).-.(
         '/|||\`
           '|`
```

Reply:
Sure thing, let me update it 👍